perf-event/examples/locality.rs

```diff
@@ -104,30 +109,17 @@ fn measure(label: &str, task: impl FnOnce()) {
         })
         .build()
         .expect("building read_miss_counter is ok");
-    let prefetch_counter = Builder::new()
-        .group(&mut group)
-        .kind(Cache {
-            which: WhichCache::L1D,
-            operation: CacheOp::PREFETCH,
-            result: CacheResult::ACCESS,
-        })
-        .build()
-        .expect("building prefetch_counter is ok");
-
```

Comment:
Is it worth just making this optional based on if the kernel returned `ENOPNOTSUPP`?

Something like this?
```
    let prefetch_counter = match Builder::new()
        .group(&mut group)
        .kind(Cache {
            which: WhichCache::L1D,
            operation: CacheOp::PREFETCH,
            result: CacheResult::ACCESS,
        })
        .build()
     {
         Ok(counter) => Some(counter),
         // The CPU does not support reading prefetches via a counter
         Err(e) if e.raw_os_error() == Some(libc::EOPNOTSUPP) => None,
         Err(e) => panic!("failed to build prefetch_counter: {e}")
     };
```

This might make the example too complex but I think it's a reasonable thing to show how to do.